packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -46,6 +46,25 @@ typedef void SemanticsAnnotator(SemanticsNode semantics);
 /// Used by [SemanticsNode.visitChildren].
 typedef bool SemanticsNodeVisitor(SemanticsNode node);
 
+/// A tag for a [SemanticsNode].
+///
+/// Tags can be interpreted by the parent of a [SemanticsNode]
+/// and depending on the presence of a tag the parent can for example decide
+/// how to add the tagged note as a child.
+///
+/// As an example, the [RenderSemanticsGestureHandler] uses tags to determine
+/// if a child node should be excluded from the scrollable area for semantic
+/// purposes.
```

Comment:
add a paragraph that talks about how if you use the `const` constructor in two places with the same _literal_ string, they will result in the same instance being created, but that if you use the `new` constructor, then instances will only be equal to themselves, even if they have the same name as another instance (including a const instance).